test/cli-test.js

```diff
@@ -9,33 +9,46 @@ var moduleVersion = require('../package').version;
 
 var exec = require('child_process').exec;
 
+function changeForOS(command) {
+
+  if(process.platform === 'win32') {
+    return command.
+      replace(/bin\/mustache/g, 'node bin\\mustache').
+      replace(/cat /g, 'type ').
```

Comment:
I think it would be preferable to wrap the regexes with `\b` symbols, like `/\bcat\b/`. Right now any word/filename ending with `cat` will instead end with `type`. It will likely not happen, but it's not as clean, I feel.